Makefile

```diff
@@ -240,12 +262,14 @@ generate-go: $(TOOLBIN)/controller-gen $(TOOLBIN)/conversion-gen  $(TOOLBIN)/moc
 ## --------------------------------------
 ## Docker
 ## --------------------------------------
+.PHONY: set-image
+set-image: $(TOOLBIN)/kustomize
+	@echo "updating kustomize image patch file for kube-app-manager resource"
+	cd config/kube-app-manager && $(TOOLBIN)/kustomize edit set image kube-app-manager=$(CONTROLLER_IMG)
 
 .PHONY: docker-build
-docker-build: test $(TOOLBIN)/kustomize ## Build the docker image for kube-app-manager
+docker-build: set-image test $(TOOLBIN)/kustomize ## Build the docker image for kube-app-manager
```

Comment:
Image is set only when it's a custom image, otherwise it's already configured in the versions overlay